### PR TITLE
#5 Migrate sample 01 no-treeshaking-material-ui to ES6

### PR DESCRIPTION
**Pull request by igoralvarez**
Added sample 01 no-treeshaking-material-ui using ES6

### CODE REVIEW

File: ES6/01 no-treeshaking-material-ui/readme.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    this.state = {
+      userName: 'defaultUserName',
+    };
+
+    this.setUsernameState = this.setUsernameState.bind(this);
+  }
+
+  setUsernameState(event) {
+    // If the state gets more complex we should use object.assign
```
Comment (brauliodiez) [inline ES6/01 no-treeshaking-material-ui/readme.md]: Remove this line, (my fault old comment that makes no sense)

File: ES6/01 no-treeshaking-material-ui/src/app.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    this.state = {
+      userName: 'defaultUserName',
+    };
+
+    this.setUsernameState = this.setUsernameState.bind(this);
+  }
+
+  setUsernameState(event) {
+    // If the state gets more complex we should use object.assign
```
Comment (brauliodiez) [inline ES6/01 no-treeshaking-material-ui/src/app.jsx]: Remove this comment

**Comment by brauliodiez** [comment]
Can you create a readme_es version as well? thanks

**Comment by igoralvarez** [comment]
Fixed and created the readme_es version.
Thanks.

**Pull request merged**